packages/device_info_plus/device_info_plus/lib/src/device_info_plus_web.dart

```diff
@@ -30,7 +31,7 @@ class DeviceInfoPlusWebPlugin extends DeviceInfoPlatform {
           'appVersion': _navigator.appVersion,
           'deviceMemory': _navigator.deviceMemory,
           'language': _navigator.language,
-          'languages': _navigator.languages,
+          'languages': _navigator.languages.toDart,
```

Comment:
Is this change maybe unrelated to the rest of the PR?

Reply:
No, this is needed for webasm compilation